componentapiexample/src/main/java/net/gini/android/vision/component/GiniVisionAppCompatActivity.java

```diff
@@ -596,22 +584,23 @@ private void showCameraOverlays() {
     }
 
     private void showExtractions(net.gini.android.models.Document giniApiDocument,
-            Map<String, SpecificExtraction> extractions) {
+            Map<String, SpecificExtraction> extractions, Document document) {
         LOG.debug("Show extractions");
-        // We display only the Pay5 extractions: paymentRecipient, iban, bic, amount and
-        // paymentReference
-        if (pay5ExtractionsAvailable(extractions)) {
+        // If we have no Pay 5 extractions we query the Gini Vision Library
+        // whether we should show the the Gini Vision No Results Screen
+        if (hasNoPay5Extractions(extractions.keySet())
```

Comment:
Component API: This is where we check the extractions and show the No Results Screen if allowed.